src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -203,7 +204,8 @@ function mergeStandByTransactionsInRange({
     return inRange || newTransaction || veryOldTransaction
   })
   const deduplicatedTransactions = deduplicateTransactions([...transactions, ...standByInRange])
-  const transactionsFromAllowedNetworks = deduplicatedTransactions.filter((tx) =>
+  const sortedTransactions = sortTransactions(deduplicatedTransactions)
```

Comment:
This was also one of the issues as initial persisted feed appeared unsorted therefore also triggering the rearrangement.